File: Backend/Backend/Modelos/DTOs/RegistrarUsuarioDTO.cs
namespace Backend.Modelos.DTOs;

public class RegistrarUsuarioDTO
{
    public string Nombres { get; set; } = null!;
    public string Apellidos { get; set; } = null!;
    public string DocumentoIdentidad { get; set; } = null!;
    public string? Telefono { get; set; }
    public string? Email { get; set; }
    public int? Edad { get; set; }


    public Usuario Mapear()
    {
        return new Usuario
        {
            Nombres = this.Nombres,
            Apellidos = this.Apellidos,
            DocumentoIdentidad = this.DocumentoIdentidad,
            Telefono = this.Telefono,
            Email = this.Email,
            Edad = this.Edad
        };
        
    }
    
}

File: frontend/src/app/Models/registrar-usuario.dto.ts
export class RegistrarUsuarioDto {
  nombres: string;
  apellidos: string;
  documentoIdentidad: string;
  telefono: string | null;
  email: string | null;
  edad: number | null;

  constructor() {
    this.nombres = '';
    this.apellidos = '';
    this.documentoIdentidad = '';
    this.telefono = null;
    this.email = null;
    this.edad = null;
  }
}
